app/views/pages/help/_processes.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="row section-content">
+  <div class="small-12 column">
+    <h3 id="processes" data-magellan-target="processes">
+      <%= t("pages.help.processes.title") %>
+    </h3>
+
+    <p>
+      <%= t("pages.help.processes.description",
+           link: link_to(t("pages.help.processes.link"), legislation_processes_path)).html_safe %>
+    </p>
+    <ul class="features">
+      <li>
+        <%= t("pages.help.processes.feature",
```

Comment:
[i see wat you did there](https://www.taleas.com/images/memes/i-see-what-you-did-there.png)

Reply:
Changes done! 😉 👍